data/library/src/main/java/com/into/websoso/data/library/LibraryRepository.kt

```
@@ -0,0 +1,15 @@
+import com.into.websoso.data.library.datasource.LibraryLocalDataSource
+import com.into.websoso.data.library.datasource.LibraryRemoteDataSource
+import javax.inject.Inject
+
+class LibraryRepository
+    @Inject
+    constructor(
+        private val libraryRemoteDataSource: LibraryRemoteDataSource,
+        private val libraryLocalDataSource: LibraryLocalDataSource,
+    ) {
+        fun fetchLibrary() {
+        }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)